phoenix-core/src/main/java/org/apache/phoenix/util/ScanUtil.java

```diff
@@ -1295,6 +1295,32 @@ public static void setScanAttributesForPhoenixTTL(Scan scan, PTable table,
         }
     }
 
+    public static Long getRPCReadTimeout(ReadOnlyProps props) {
+        if (props.get(HConstants.HBASE_RPC_READ_TIMEOUT_KEY) != null) {
+            return props.getLong(HConstants.HBASE_RPC_READ_TIMEOUT_KEY,
+                    HConstants.DEFAULT_HBASE_RPC_TIMEOUT);
```

Comment:
Since RPC timeout is in int, we could do this too `(long) props.getInt(HConstants.HBASE_RPC_READ_TIMEOUT_KEY, HConstants.DEFAULT_HBASE_RPC_TIMEOUT)` 

But there doesn't seem to be that much diff in time complexity of Long#parseLong vs Integer#parseInt (specifically for string values that are not int overflows, which is always the case for us). We should be good.

Reply:
Even hbase makes this confusing, at some places this is being read as int and some places as long, should have kept same to avoid confusions :)